### ISSUE TITLE
Copy Row wont paste sequence to sequence

### ISSUE THREAD

**Issue by dwbarnett** [smeighan/xLights#650]
With 2 XL open. If using copy row in one sequencer, then trying to paste row in other open sequence; ie copy from sequence to sequence.  The Paste Row is grayed out in the second sequencer.

Workaround,  Ctrl C, Ctrl V will work, after high-lighting the row.

The manual, states you can Copy/Paste Row sequence to sequence; ref page:Ch4 pg 80 states:

Then move to where you want to paste the effects (in the same sequence
or another sequence), right click against the model name and select Paste
Row. The data copied remains in the clipboard buffer. The Paste
operation does not have to be immediately after the Copy operation.

**Comment by AzGilrock** [comment]
The code is not changing for this so they can update the manual.  There is no way for one copy of xLights to know what another copy of xLights is doing to be able to activate menu options.  The program is not designed to work with 2 copies open so whatever happens to work is by luck.

**Comment by AzGilrock** [comment]
Really you just need to trick the program into thinking it has data to paste.  So do a Copy Row in sequence 1.  Then do a Copy Row in sequence 2.  Go back to sequence 1 and do a Paste Row.  That should allow it to work.  Then the menu will remain active and you can continue to do copies from one sequence and pastes into the other.

**Comment by dwbarnett** [comment]
Gil, still didn't work. Rich Renn and I both tried the above steps. It will work if you use copy row, then use Ctrl V to paste. So, still a workaround, from the manual statement.

**Comment by AzGilrock** [comment]
Ok thanks for testing it.  I figured out a way to make the row copy/paste work no matter what version of xlights it was copied from and also fixed it not doing a row paste if you happened to have Paste by Cell active which shouldn't matter for a row paste.   By the way the current version also had a bug in the row copy/paste I fixed yesterday but I think it only affects you if you've scrolled vertically because it doesn't subtract off the number of rows that are hidden.
